Contributions/Using_Linux_in_a_Safe_System.md

```diff
@@ -0,0 +1,348 @@
+  functionality will not be interrupted.
+
+  Clarifying the scope of the design goes a long way in defining criteria for
+  identifying acceptable solutions.
+
+* **Linux is intrinsically not safe**
+
+  While Linux has many remarkable properties, safety is not one of them, which means
+  that it won't be possible to just allocate safety requirements to Linux, and expect
+  them to be automatically satisfied. Quite the opposite, actually.
+
+  Having to cope with safety requirements assigned to Linux will involve a price to
+  be paid, in one form of another.
+
+  The problem with said cost is that it might *also* affect negatively the goals
+  that led to the selection of Linux, in the first place. For example, if Linux was
+  selected because of its a rapid turnaround in fixing security bugs, selecting
+  an approach that would introduce an lengthy qualification process would undermine
+  the advantage of the rapid turnaround.
+
+* **Aggressively placing safety requirements on anything but (components of) the Linux kernel**
+
+  The natural consequence of the previous point is that one should explore - given a
+  specific set of safety requirements for the system as a whole, if they could be
+  satisfied through some component other than Linux (more on this later).
+
+  There are several additional reasons for favouring an external component, besides
+  what has been mentioned already:
+  *  much greater probability for the component to be simpler (and thus easier to qualify)
+  *  opportunity to have a better control over the component (e.g. use a software
+     architecture that is better suited for safety, formalised processes, etc.)
+  *  decouple release cycles of the Linux kernel from the safety component
+  *  other non-technical reasons (e.g. different licensing)
+
+## **Examples of Safety-oriented System Design Concepts**
+
+**Premise:** applying the initial consideration from the previous sections, it is
+necessary to define what is required at system level in terms of both availability
+and integrity.
+
+The simpler - and most common - case is to require detection of interference
+affecting components that are recipient of safety requirements.
+
+Interference affecting a component with safety requirements must be both detected
+and reported, in a way that is compatible with the associated time requirements.
+At this point, appropriate action(s) will be taken, in accordance with the
+pre-established policy, to ensure adequate availability
+(e.g. restarting, onlining of backup system, etc.)
+
+As mentioned earlier, guaranteeing availability can be much more difficult
+and/or costly.
+
+* **Application Monitoring**
+
+  Monitoring is a simple approach to detection of interference that affects
+  component with safety requirement.
+  
+  It can save from extensive hardening.
+
+  However two conditions must be met:
+  1. It must be possible to express a simple check, in a way that can repeatedly,
```

Comment:
express a simple check is not clear to me.
The check can be complex, I would say must be rigorous. Simple or complex are subjectives.
Are you referring to the output of the check that should be simple?

Reply:
I think "simple" is redundant here: the important aspects are that it can be consistently and reliably performed, and that it produces an unequivocal result.